ansible/roles/baselayout/vars/main.yml

```diff
@@ -146,10 +146,10 @@ packages: {
   ],
 
   ubuntu: [
-    'ccache,g++,gcc,g++-6,gcc-6,git,libfontconfig1,sudo,python3-pip',
+    'ccache,g++,gcc,g++-6,gcc-6,git,libfontconfig1,sudo,python3-pip,libcap-dev',
   ],
 
   ubuntu1404: [
-    'ntp,gcc-8,g++-8,gcc-6,g++-6,g++-4.8,gcc-4.8,g++-4.9,gcc-4.9,binutils-2.26',
+    'ntp,gcc-8,g++-8,gcc-6,g++-6,g++-4.8,gcc-4.8,g++-4.9,gcc-4.9,binutils-2.26,libcap-dev',
```

Comment:
We don't build `master`/16.x on Ubuntu 14.04 so we do not need to install the new package there.
https://github.com/nodejs/build/blob/371bb7626abe773a56c7c62e76d0a551ba1b9717/jenkins/scripts/VersionSelectorScript.groovy#L33-L34